lib/check-started.js

```diff
@@ -1,16 +1,14 @@
 module.exports = checkStarted;
 
 var request = require('request').defaults({json: true});
-var getSeleniumHub = require('./get-selenium-hub.js');
+var getSeleniumStatusUrl = require('./get-selenium-status-url.js');
 
 function checkStarted(seleniumArgs, cb) {
   var retries = 0;
-
-  var hub = getSeleniumHub(seleniumArgs);
-
-  var retryInterval = 200;
+  var hub = getSeleniumStatusUrl(seleniumArgs);
   // server has one minute to start
-  var maxRetries = 60 * 1000 / retryInterval;
```

Comment:
no need to microptimize this, otherwise you have to do the computation to know how much time you have to start